ocropus-econf

```diff
@@ -84,11 +84,11 @@ for fname,e,t,m,c in outputs:
 if perfile is not None: perfile.close()
 if allconf is not None: allconf.close()
 
-sys.stderr.write("errors    %8d\n"%errs)
-sys.stderr.write("missing   %8d\n"%missing)
-sys.stderr.write("total     %8d\n"%total)
-sys.stderr.write("err       %8.3f %%\n"%(errs*100.0/total))
-sys.stderr.write("errnomiss %8.3f %%\n"%((errs-missing)*100.0/total))
+print "errors    %8d"%errs
```

Comment:
Use `print()` and be Python 3 ready. There's only one argument, so there's no effect on Python 2.